go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/neovim/go-client
+
+go 1.12
```

Comment:
I don't see this in https://github.com/golang/go/wiki/Modules#example . Does this mean we require 1.12? Should it be 1.11 instead?

Do we need a `require()` section?

Reply:
just remove it.

> Do we need a require() section?

The `require()` section needs if this repo depends on another third-party package. 
neovim/go-client written in only Go standard package. In this case, we don't need a `require`  section.